http/getAxiosConfig.ts

```diff
@@ -4,6 +4,26 @@ import { getHubSpotApiOrigin } from '../lib/urls';
 import { AxiosConfigOptions } from '../types/Http';
 import { CLIConfig } from '../types/Config';
 import { AxiosRequestConfig } from 'axios';
+import https from 'https';
+import http from 'https';
+
+// Total number of sockets across all hosts
+const MAX_TOTAL_SOCKETS = 25;
+
+// Total number of sockets per each host
+const MAX_SOCKETS_PER_HOST = 5;
```

Comment:
This effectively limits the max number of of inflight API requests to a particular domain to 5.  If 5 requests are in flight, they will queue.  This value was just the value I found in my testing that seemed to greatly reduce the `ENOMEM` errors.  Instead of roughly 5%-10% `hs project upload` commands failing, this got it down to under 1% for me (I made it to 250 uploads before getting a `ENOMEM`)